snyk/managers.py

```diff
@@ -296,6 +296,16 @@ def all(self) -> Any:
 
 
 class IssueSetManager(SingletonManager):
+    def _convert_reserved_words(self, data):
+        if "issues" in data:
+            if "vulnerabilities" in data["issues"]:
```

Comment:
Wouldn't this be nicer in a 1 liner?
```suggestion
            if "issues" in data and "vulnerabilities" in data["issues"]:
```

Reply:
Thanks, fixed.